apps/kvarteret/components/TodayItem.tsx

```diff
@@ -1,4 +1,7 @@
-const TodayItem = ({ event, bold = false }) => {
+import { Event } from "dak-components/lib/cms/queries/events";
+import { CrescatEvent } from "dak-components/lib/crescat";
+
+const TodayItem = ({ event, bold = false }: { event: { time: String; room: String; title: String; }; bold?: boolean }) => {

```

Comment:
```suggestion
const TodayItem = ({ event, bold = false }: { event: { time: String; room: String; title: String; }; bold?: boolean }) => {

```
imports not used I think

Reply:
Is this still relevant?

Reply:
as long as they are greyed out in vscode then yes they are not needed. And works